packages/expo-updates/ios/EXUpdates/Update/LegacyUpdate.swift

```diff
@@ -46,10 +46,10 @@ internal final class LegacyUpdate: Update {
 
     var runtimeVersion: String
     let manifestRuntimeVersion = manifest.runtimeVersion()
```

Comment:
fwiw, we require it to be a string on android:
https://github.com/expo/expo/blob/main/packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/LegacyUpdateManifest.kt#L117
https://github.com/expo/expo/blob/main/packages/expo-manifests/android/src/main/java/expo/modules/manifests/core/LegacyManifest.kt#L16

It seems on iOS we allow it to be anything for some reason. Let's follow-up this PR with one to ensure it's a nullable string on iOS as well.

That being said, I don't think there should be a case where it is something other than a string or null. What steps produce a non-null object other than a string?